casper/src/main/scala/coop/rchain/casper/Estimator.scala

```diff
@@ -127,11 +127,12 @@ final class Estimator[F[_]: Sync: Log: Metrics: Span](
   ): F[Map[BlockHash, Long]] = {
     def hashParents(hash: BlockHash, lastFinalizedBlockNumber: Long): F[List[BlockHash]] =
       for {
-        currentBlockNumber <- blockDag.lookup(hash).map(_.get.blockNum)
-        result <- if (currentBlockNumber < lastFinalizedBlockNumber)
-                   List.empty[BlockHash].pure[F]
-                 else
-                   blockDag.lookup(hash).map(_.get.parents)
+        block              <- blockDag.lookupUnsafe(hash)
```

Comment:
Lets revert formatting as well, to not make an impression that there are changes to Estimator

Reply:
Makes sense.